containers/kubernetes/how-to/access-audit-logs.mdx

```diff
@@ -0,0 +1,71 @@
+  validation: 2024-01-23
+  posted: 2024-01-23
+categories:
+  - kubernetes
+---
+
+Kubernetes Kapsule and Kosmos control plane metrics and logs are integrated into Cockpit, providing you with a centralized hub for monitoring the control plane, nodes, managed resources, and cluster system applications.
+While this initial integration empowers you with the autonomy to troubleshoot issues promptly, we have taken the next stride by extending the functionality—now, Kubernetes audit records are also exported into Cockpit.
+
+Kubernetes audit logs provide you detailed access to the activities generated by users, by applications that use the Kubernetes API, and by the control plane itself.
+Auditing allows cluster administrators to answer the following questions:
+
+* what happened?
+* when did it happen?
+* who initiated it?
+* on what did it happen?
+* where was it observed?
+* from where was it initiated?
+* to where was it going?
+
+Audit logging in Kubernetes clusters now by default on for clusters with dedicated control planes. Use audit logging to keep a chronological record of calls made to the Kubernetes API server, investigate suspicious API requests, collect statistics, or create monitoring alerts for unwanted API calls. 
+All logs are centralized in [Cockpit](/observability/cockpit/quickstart/) for free.
+
+<Message type="tip">
+    Monitoring calls to the `kube-apiserver` is a matter of security compliance and perhaps a hard requirement for some of the certifications you are keen to obtain.
+</Message>
+
+## How to enable audit logging
+
+You can enable audit logging for all your clusters that use a dedicated control plane. If you have created your cluster before the introdution of the feature, you have to enable it for the cluster from your Scaleway console.
+
+<Message type="tip">
+    Audit logs are activated by default for all new clusters using dedicated control planes.
+</Message>
+
+1. [Activate Cockpit](/observability/cockpit/how-to/activate-cockpit/) for the project your Kubernetes cluster is located in.
+2. Enable **Audit logs** in the **Security and Compliance** section of your Kubernetes cluster **Settings** tab.
+2. [Access Grafana](/observability/cockpit/how-to/access-grafana-and-managed-dashboards/) to view the logs in the **Kubernetes Cluster Audit Logs** dashboard within Cockpit, as well as in the **Explore** section of Cockpit/Grafana.
+3. Finally, examine the [Kubernetes Audit Policy](https://gist.github.com/Tomy2e/938e85ab78325e6a42155d54e1df8d1b) file, which contains a list of rules, giving you complete visibility into our API server configuration and the chosen request treatments or exclusions.
+
+<Message type="tip">
+    In case of doubt if audit logs for your cluster are enabled, you can verify the status of the feature from the clusters **Security** tab at any time.
+</Message>
+
+## How to disable audit logging
+
+In case you do not want the audit logs of your cluster being stored, you can disable the feature.
```

Comment:
The rationale behind disabling audit log could be to save memory on not sensitive clusters.
The official k8s doc states: 
_The audit logging feature increases the memory consumption of the API server because some context required for auditing is stored for each request._
https://kubernetes.io/docs/tasks/debug/debug-cluster/audit/